python/tests/core/metrics/test_metrics.py

```diff
@@ -42,6 +46,41 @@ def test_distribution_metrics_series() -> None:
     assert dist.variance == data.var()
 
 
+def test_distribution_variance_m2() -> None:
+    import statistics
+
+    dist_list = DistributionMetric.zero(MetricConfig())
+    dist_pandas = DistributionMetric.zero(MetricConfig())
+    dist_numpy = DistributionMetric.zero(MetricConfig())
+    test_input = [1, 2, 3, 4]
+
+    list_test_input = PreprocessedColumn()
+    list_test_input.list.ints = test_input
+    n = len(test_input)
+    mean = sum(test_input) / n
+    variance = statistics.variance(test_input)
```

Comment:
should comment the DoF used here just for clarity

Reply:
The next line shows the normalization in the m2 assignment, and the statistics.variance doesn't have a DoF parameter, but I added a comment pointing out that this is a sample variance calculated in statistics module https://docs.python.org/3/library/statistics.html#statistics.variance